docs: add client configuration section and support matrix for AI Bridge

matifali opened the pull request:
Revised the AI Bridge documentation to clarify provider configuration steps, including new sections for OpenAI, Anthropic, and Amazon Bedrock. Added details on pre-configuring templates and using AI Bridge with Coder Tasks. Introduced a new image illustrating the AI Bridge implementation details. Updated compatibility notes for various AI clients.

Addresses coder/aibridge#4

Preview: https://coder.com/docs/@aibridge-client-instructions/ai-coder/ai-bridge

matifali committed:
docs(ai-bridge): update configuration instructions and add support for additional providers

Revised the AI Bridge documentation to clarify provider configuration steps, including new sections for OpenAI, Anthropic, and Amazon Bedrock. Added details on pre-configuring templates and using AI Bridge with Coder Tasks. Introduced a new image illustrating the AI Bridge implementation details. Updated compatibility notes for various AI clients.

Addresses coder/aibridge#4

matifali committed:
Merge branch 'main' into aibridge-client-instructions

matifali committed:
Update AI Bridge documentation table

Revised client support table for enhanced clarity.

david-fraley commented on docs/ai-coder/ai-bridge.md:
```suggestion
Bridge can relay traffic to other OpenAI- or Anthropic-compatible services or model proxies like LiteLLM by pointing the base URL variables above at the provider you operate. Share feedback or follow along in the [`aibridge`](https://github.com/coder/aibridge) issue tracker as we expand support for additional providers.
```

Changed the last sentence to not call out "stabilizes". It implies that it's currently unstable and I'm not a fan of that implication

david-fraley commented on docs/ai-coder/ai-bridge.md:
```suggestion
The table below shows tested AI clients and their compatibility with AI Bridge. Click each client name for vendor-specific configuration instructions. Report issues or share compatibility updates in the [aibridge](https://github.com/coder/aibridge) issue tracker.
```
Changed up the phrasing to be more assertive/we've actually done the work.

david-fraley reviewed (changes requested):
If you've got time to address this high level feedback before release, let's do it. If not, let's make it happen quickly after. It's not a hard requirement for merge, but I don't want to let it hang out too long... 

This page is getting MaSSiVe 😂. Can we break it up into a top-level "AI Bridge" w/ the summary, then sub pages covering (this is just a first stab, feel free to riff on the sub page breakdown)
* Setup
* Client Config
* MCP
* Monitoring
* Reference (super technical documentation)

matifali committed:
docs: address feedback on AI Bridge documentation

Splits the AI Bridge documentation into multiple pages to improve readability and navigation.

matifali commented:
I have addressed the feedback and pushed the changes.

matifali committed:
docs: address language and sentence suggestions for AI Bridge

matifali committed:
Update AI Bridge for consistent terminology

Use "AI Bridge" consistently across the documentation, replacing instances of "Bridge." Adjust headings and content throughout to ensure clarity and alignment with the AI Bridge branding.

matifali committed:
Merge branch 'main' into aibridge-client-instructions

matifali committed:
Improve AI Bridge documentation

- Capitalize section headings for consistency.
- Use the singular form for "provider" for clarity.
- Ensure document syntax matches Coder documentation style guidelines.

david-fraley commented on docs/ai-coder/ai-bridge/index.md:
```suggestion
```

david-fraley commented on docs/manifest.json:
```suggestion
							"description": "How to set up and configure AI Bridge",
```

david-fraley commented on docs/manifest.json:
```suggestion
							"description": "How to use AI Bridge with MCP",
```

david-fraley commented on docs/manifest.json:
```suggestion
							"description": "How to monitor AI Bridge",
```

david-fraley committed:
Update docs/ai-coder/ai-bridge/index.md

david-fraley committed:
Update docs/manifest.json

david-fraley committed:
Update docs/manifest.json

david-fraley committed:
Update docs/manifest.json

david-fraley committed:
Update docs/manifest.json

david-fraley committed:
Update docs/manifest.json

dannykopping commented on docs/ai-coder/ai-bridge/index.md:
```suggestion
- Uncover high-leverage use cases
```

dannykopping commented on docs/ai-coder/ai-bridge/index.md:
```suggestion
```

dannykopping commented on docs/ai-coder/ai-bridge/setup.md:
```suggestion
AI Bridge runs inside the Coder control plane (`coderd`), requiring no separate compute to deploy or scale. Once enabled, `coderd` runs the `aibridged` in-memory and brokers traffic to your configured AI providers on behalf of authenticated users.
```

dannykopping reviewed (commented):
Really liking the reorganisation.

Pausing here for the moment; ran out of time to review.
I've only reviewed up to the start of Client Config page, in the order of how it appears on the website.

Thanks for providing a docs preview link, that was very helpful.

dannykopping commented on docs/ai-coder/ai-bridge/mcp.md:
I think we should make this a code block. The docs get rendered a bit weirdly with a block quote:

<img width="632" height="190" alt="image" src="https://github.com/user-attachments/assets/839ff7c7-0e45-4de2-838c-a48eec49be1b" />

david-fraley committed:
Update docs/ai-coder/ai-bridge/index.md

Co-authored-by: Danny Kopping <danny@coder.com>

david-fraley committed:
Update docs/ai-coder/ai-bridge/index.md

Co-authored-by: Danny Kopping <danny@coder.com>

david-fraley committed:
changes to index

david-fraley committed:
add next steps section

david-fraley committed:
cleaned up index owrkding

david-fraley committed:
Update docs/ai-coder/ai-bridge/setup.md

Co-authored-by: Danny Kopping <danny@coder.com>

david-fraley committed:
setup page udpates

david-fraley committed:
changes to client-config

david-fraley committed:
Update docs/ai-coder/ai-bridge/client-config.md

Co-authored-by: Danny Kopping <danny@coder.com>

david-fraley committed:
Update docs/ai-coder/ai-bridge/client-config.md

Co-authored-by: Danny Kopping <danny@coder.com>

david-fraley committed:
Update docs/ai-coder/ai-bridge/client-config.md

Co-authored-by: Danny Kopping <danny@coder.com>

david-fraley commented on docs/ai-coder/ai-bridge/client-config.md:
I'll leave this one for @matifali to quick follow

david-fraley committed:
more updates to client-config

david-fraley committed:
add code block

david-fraley committed:
Update docs/ai-coder/ai-bridge/mcp.md

Co-authored-by: Danny Kopping <danny@coder.com>

david-fraley committed:
remove mermaid diagram

david-fraley committed:
Merge branch 'main' into aibridge-client-instructions

david-fraley committed:
linting stuff

david-fraley committed:
lint

matifali commented on docs/ai-coder/ai-bridge/mcp.md:
This is a bug in our docs engine.
